### PR TITLE
Set dependency versions to released tags

### PR DESCRIPTION
**Pull request by danielpieczko**
Pin the versions of all the dependent libraries that have been released so far.

### CODE REVIEW

File: lib_xua/lib_build_info.cmake [hunk 1]
```diff
@@ -20,14 +20,14 @@ set(LIB_INCLUDES api
...
                  src/midi)
 set(LIB_OPTIONAL_HEADERS xua_conf.h static_hid_report.h)
-set(LIB_DEPENDENT_MODULES "lib_locks"
-                          "lib_logging"
-                          "lib_mic_array(feature/xcommon_cmake)"
+set(LIB_DEPENDENT_MODULES "lib_adat(1.1.0)"
+                          "lib_locks(2.2.0)"
+                          "lib_logging(3.2.0)"
+                          "lib_mic_array(4.6.0)"
                           "lib_spdif"
```
Comment (ed-xmos) [inline lib_xua/lib_build_info.cmake:27]: this is v5.0.0 in module_build_info

File: lib_xua/lib_build_info.cmake [hunk 2]
```diff
@@ -20,14 +20,14 @@ set(LIB_INCLUDES api
...
-                          "lib_xassert"
-                          "lib_xud"
-                          "lib_adat"
-                          "lib_sw_pll")
+                          "lib_sw_pll(2.1.0)"
+                          "lib_xassert(4.2.0)"
+                          "lib_xud")
```
Comment (ed-xmos) [inline lib_xua/lib_build_info.cmake:30]: 2.2.3 in module_build_info. Any reason for the diff?

Comment (danielpieczko) [reply lib_xua/lib_build_info.cmake:30]: lib_xud hasn't been released yet with xcommon_cmake support for this upcoming release. Then both module_build_info and lib_build_info.cmake will be 2.3.0.

File: lib_xua/lib_build_info.cmake (hunk 1 above)
Comment (danielpieczko) [reply lib_xua/lib_build_info.cmake:27]: Will update both to 6.x.y when lib_spdif is tagged for the upcoming release.

**Review by ed-xmos — APPROVED** [review]
Looks good.

**Pull request merged**